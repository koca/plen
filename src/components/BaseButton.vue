<template>
  <button v-on="$listeners" v-bind="$attrs" class="btn-base sm:hover:bg-dark focus:outline-none px-3 u-align-center h-full">
    <slot></slot>
  </button>
</template>
<script>
export default {};
</script>
