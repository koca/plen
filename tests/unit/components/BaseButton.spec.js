import { shallowMount } from "@vue/test-utils";
import BaseButton from "@/components/BaseButton.vue";

describe("BaseButton.vue", () => {
  it("renders properly", () => {
    let wrapper;
    wrapper = shallowMount(BaseButton);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
