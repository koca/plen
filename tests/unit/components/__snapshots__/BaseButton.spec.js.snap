// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton.vue renders properly 1`] = `<button class="btn-base sm:hover:bg-dark focus:outline-none px-3 u-align-center h-full"></button>`;
